backend/src/main.ts

```diff
@@ -6,7 +6,10 @@ async function bootstrap() {
   const app = await NestFactory.create(AppModule);
 
   app.enableCors({
-    origin: 'http://localhost:3000',
+    origin: [
+      'http://localhost:3000',
+      'https://taskman-krpexpd5u-kins-projects-a2930a32.vercel.app',
```

Comment:
please create env variable for these.
like
```
LOCAL_BE_URL=http://localhost:3000
SERVER_BE_URL=https://taskman-krpexpd5u-kins-projects-a2930a32.vercel.app
```

also use the env variables for fetching data, basically if NODE_ENV set to development you use LOCAL_BE_URL  if production use SERVER_BE_URL as your base url for fetching data from BE on FE

also please create 2 file for env.

```
.env   // this stores all env variable with its values
.env.example  // this stores the same env variable but without values, its for setting up local for other developers
```